content/outbreaks/_index.en.md

```diff
@@ -4,6 +4,94 @@ categories: ["R"]
 tags: ["wiki"]
 weight: 10
 ---
+![msfepimeasles](images/msfepimeasles.jpg?width=30pc)
+
+**Source**: Elise Mertens/MSF; MSF is working with the Ministry of Public Health throughout the year to monitor areas at risks of disease outbreaks. Week after week Louis, the MSF epidemiologist, monitors the health zones that have reported cases and the levels of epidemic alert. He is also in charge of coordinating the post-vaccination coverage survey to ensure that the vaccination coverage rate is sufficient.
+
+The outbreak templates have been developed to cover four of the most common outbreak scenarios MSF teams face in our project locations. They include:
+
+1. **Cholera/Acute watery diarrhea**
+2. **Meningitis**
+3. **Measles/Rubella**
+4. **Acute Jaundice Syndrome (often suspected to be Hepatitis E)**
+
+Some recent examples of such outbreak responses and the data analysis that was done during those outbreaks is available here:
+
+* [Cholera epidemic in Yemen, 2016–18: an analysis of surveillance data](https://www.thelancet.com/journals/langlo/article/PIIS2214-109X(18)30230-4/fulltext)
+* [Invasive Meningococcal Meningitis Serogroup C Outbreak in Northwest Nigeria, 2015 - Third Consecutive Outbreak of a New Strain](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4958021/?report=reader)
+* [Lessons and Challenges for Measles Control from Unexpected Large Outbreak, Malawi](https://wwwnc.cdc.gov/eid/article/19/2/12-0301_article)
+* [A large outbreak of Hepatitis E virus genotype 1 infection in an urban setting in Chad likely linked to household level transmission factors, 2016-2017](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0188240)
+
+You can access these by selecting them from the RStudio RMarkdown template menu:
+> RStudio>File>New File>R Markdownn>Template>Select Outbreak Report Template
+
+# Getting started with outbreak templates in R
+
+This section (INSERT LINK TO THE WALK THROUGH ON OUTBREAK DATA ANALYSIS) offers you a step by step guide on how to use the outbreak templates to fit your data and run the analysis.
```

Comment:
I suggest adding an explicit "TODO" so that we don't forget it.

```suggestion
This section (TODO: INSERT LINK TO THE WALK THROUGH ON OUTBREAK DATA ANALYSIS) offers you a step by step guide on how to use the outbreak templates to fit your data and run the analysis.
```